CRM/Profile/Page/MultipleRecordFieldsListing.php

```diff
@@ -35,6 +35,12 @@ class CRM_Profile_Page_MultipleRecordFieldsListing extends CRM_Core_Page_Basic {
 
   public $_contactType = NULL;
 
+  public $_customGroupId = NULL;
```

Comment:
This is protected on `CRM_Profile_Form` - which is a bit confusing but I guess the 2 are not connected

Reply:
No this routes straight through CRM_Core_Page_Basic -> CRM_Core_Page i.e. its not a form its a Page Also can't make it protected because this sets it https://github.com/civicrm/civicrm-core/blob/master/CRM/Custom/Page/AJAX.php#L113

Reply:
ok - feels like it's all a bit shonky. Don't look too hard